core/src/test/java/io/questdb/test/cairo/fuzz/DedupInsertFuzzTest.java

```diff
@@ -294,11 +319,11 @@ public void testRandomColumnsDedupMultipleKeyCol() throws Exception {
                 rnd.nextDouble(),
                 rnd.nextDouble(),
                 0.1 * rnd.nextDouble(),
-                0.1 * rnd.nextDouble(),
+                convertToParquet ? 0 : 0.1 * rnd.nextDouble(),
```

Comment:
nit: a clarifying comment/todo might be handy here as it's not clear why we set some probabilities to 0 in case of parquet.

Reply:
addressed